src/app/models/form.server.model.ts

```diff
@@ -419,27 +422,31 @@ const compileFormModel = (db: Mongoose): IFormModel => {
     return newForm
   }
 
-  // Transfer ownership of the form to another user
-  FormSchema.methods.transferOwner = async function (
-    currentOwner: IUserSchema,
-    newOwnerEmail: string,
-  ) {
-    // Verify that the new owner exists
+  FormSchema.methods.transferOwner = async function (newOwnerEmail: string) {
+    const currentOwner = await User.findById(this.admin)
+    if (!currentOwner) {
+      throw new DatabaseError('Admin of the form cannot be found')
```

Comment:
should this be a `UserNotFound` error, since the database is working correctly?

Reply:
Yea, was debating on that. I kind of want to differentiate the severity of the error also, since if the form has an admin but the admin is no longer in the database, it should be a big problem since that should never happen.

What do you think? Still change to `UserNotFoundError`?